data/saas/config/saas_example_config.yml

```diff
@@ -254,3 +263,28 @@ saas_config:
           param_values:
             - name: email
               identity: email
+    - name: tickets
+      requests:
+        read:
+          - method: GET
+            path: /v1/tickets
+            query_params:
+              - name: status
+                value: open
+            param_values:
+              - name: customer_id
+                references:
+                  - dataset: saas_connector_example
+                    field: customer.id
+                    direction: from
+          - method: GET
+            path: /v1/tickets
+            query_params:
+              - name: status
+                value: closed
```

Comment:
I see this status use case a few times in our legacy adaptors, if we followed the legacy approach, we would have different endpoints per identity or per status in this example `users_by_email`, `users_by_phone`, `open_tickets`, `closed_tickets`, etc. I didn't like this approach since the end result of these different endpoints is still just `user` and `ticket` entities. I thought of potentially changing the way we define and use query params but that seemed like a complex change. This request list approach was the simplest I could think of right now.

Reply:
makes sense @galvana, thanks for writing it out!